pkg/block/fetcher.go

```diff
@@ -170,6 +170,51 @@ func DefaultModifiedLabelValues() [][]string {
 	}
 }
 
+// Fetcher interface to retieve blockId information from a bucket.
+type BlockIDsFetcher interface {
+	//Get active blocksIds returning it via channel (streaming) and response.
```

Comment:
I think it is worth documenting what does active block means in the method name. Does it mean anything here?
If there is no special meanings, let's rename the method to `GetBlockIDs`. No need to mention partial block IDs in the method name itself. Just document what should we return and how should we set the response.

Reply:
Yes. The Active block here means block with meta.json, while the partial block means block without meta.json. I'll add those to the interface method description.